packages/contracts/src/predeploys.ts

```diff
@@ -11,11 +11,11 @@ export const predeploys = {
   OVM_L2ToL1MessagePasser: '0x4200000000000000000000000000000000000000',
```

Comment:
Should we also remove `OVM` from the L2 contracts? Or do we like that pattern?

Reply:
I was on the fence on that one but after speaking to @karlfloersch I leaned towards keeping the OVM prefix on predeploys only. Reasons are that most of them will be obsoleted soon while the ones that remain: `OVM_L2ToL1MessagePasser` and `OVM_SequencerFeeVault` make sense to be prefixed. 
Also @tynes leaned me that way as there are a lot of comments around the geth codebase that reference the predeploys with the ovm prefix. Also I _think_ he said it will break some logic in the gas oracle. 
What do you think?

Reply:
Ok let's leave it for the predeploys